website/docs/d/guest_os_customization.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+subcategory: "Virtual Machine"
+layout: "vsphere"
+page_title: "VMware vSphere: vsphere_guest_os_customization"
+sidebar_current: "docs-vsphere-data-guest-os-customization"
+description: |-
+  Provides a VMware vSphere guest customization spec data source. This can be used to apply the customization spec when virtual machine is cloned
```

Comment:
```suggestion
  Provides a VMware vSphere customization specification data source. This can be used to apply a customization specification to the guest operating system of a virtual machine after cloning.
```